packages/cms/src/components/editors/SimpleVisualizationEditor.jsx

```diff
@@ -207,8 +296,14 @@ class SimpleVisualizationEditor extends Component {
 
   render() {
     const {object, rebuildAlertOpen, payload} = this.state;
+    const {locale} = this.props;
+    const formatters = this.context.formatters[locale];
+    const formatterList = formatters ? Object.keys(formatters) : [];
```

Comment:
Please alpha-sort this list, it gets confusing when the list is rather large (I'm testing with CDC)